roles/os_firewall/tasks/firewall/firewalld.yml

```diff
@@ -1,7 +1,8 @@
 ---
 - name: Install firewalld packages
-  package: name=firewalld state=present
-  when: not openshift.common.is_containerized | bool
+  package:
+    name: firewalld
+    state: present
```

Comment:
We would never hit this code path if `is_atomic` because we already don't use firewalld if `is_atomic`.